app/.gitignore

```diff
@@ -1,3 +1,3 @@
+.lock-waf_linux2_build
```

Comment:
Was the original pattern too wide for you?

Reply:
I see this lock file sometimes and wouldn't want to see it in git.